src/electron/views/screenshot/screenshot-container.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { BoundingRectangle } from 'electron/platform/android/scan-results';
+import { HighlightBox } from 'electron/views/screenshot/highlight-box';
+import { Screenshot } from 'electron/views/screenshot/screenshot';
+import { screenshotContainer } from 'electron/views/screenshot/screenshot-container.scss';
+import { isEmpty } from 'lodash';
+import * as React from 'react';
+
+export interface ScreenshotContainerProps {
+    screenshotData?: ScreenshotData;
+    highlightBoxRectangles?: BoundingRectangle[];
+}
+
+export const ScreenshotContainer = NamedFC<ScreenshotContainerProps>('ScreenshotContainer', props => {
+    if (isEmpty(props.screenshotData)) {
+        return <div>Screenshot for scan is unavailable</div>;
+    }
+
+    return (
+        <div className={screenshotContainer}>
+            <Screenshot encodedImage={props.screenshotData.base64PngData} />
+            {renderHighlightBoxes(props.highlightBoxRectangles)}
+        </div>
+    );
+});
+
+function renderHighlightBoxes(highlightBoxRectangles: BoundingRectangle[]): JSX.Element[] {
+    if (isEmpty(highlightBoxRectangles)) {
+        return;
+    }
+
+    const renderedBoxes: JSX.Element[] = [];
+    let keyId = 1;
+
+    highlightBoxRectangles.forEach(instance => {
+        renderedBoxes.push(<HighlightBox key={keyId} boundingRectangle={instance} />);
```

Comment:
Using an index for a key id is probably acceptable for our purposes, but react is actually capable of being smarter about re-renders if we can use a key that remains stable when boxes are added/removed from the list. Probably the uid of the corresponding result would be ideal; I don't think changing the view model layout to pipe that association around is necessary for this PR, but it might be worth noting in a comment here that using the result uid would be better so we don't forget to make that update in the "interactive screenshot" feature when we'll need to start piping the UID in anyway.

Reply:
Added comment for the future interactive screenshot feature